cudamapper/tests/Test_CudamapperMatcher.cpp

```diff
@@ -10,16 +10,17 @@
 */
 
 #include "gtest/gtest.h"
+#include "cudamapper_file_location.hpp"
 #include "../src/index_cpu.hpp"
 #include "../src/index_generator_cpu.hpp"
 #include "../src/matcher.hpp"
 
 namespace claragenomics {
 
     TEST(TestCudamapperMatcher, SampleTest) {
-        IndexGeneratorCPU index_generator("data/one_read_one_minimizer.fasta", 4, 1);
-        IndexCPU index(index_generator);
-        Matcher matcher(index);
+        IndexGeneratorCPU index_generator(std::string(CUDAMAPPER_BENCHMARK_DATA_DIR) + "/one_read_one_minimizer.fasta", 4, 1);
+//        IndexCPU index(index_generator);
```

Comment:
let's remove these commented lines if they're not necessary

Reply:
I uncommented those lines as that's dummy data we're going to do the tests on once the corresponding code is ready